databricks/koalas/series.py

```diff
@@ -206,8 +206,8 @@ def getField(self, name):
                     "Field {} not found, possible values are {}".format(name, ", ".join(fnames)))
             return Series(self._scol.getField(name), self._kdf, self._index_info)
 
-    # TODO: automate the process here
     def alias(self, name):
+        """An alias for :meth:`Series.rename`."""
```

Comment:
we should also add alias to the Series doc page.

Two ways:

One is to create a Spark API section, and just put alias there. The other is to put it in the same section as rename. WDYT? cc @thunterdb

Reply:
Nothing prevents you from putting it into both sections. We will eventually have a spark section anyway.

Reply:
Yes that's a brilliant idea Tim.